app/views/admin/users/_user_form_fields.html.erb

```diff
@@ -6,11 +6,15 @@
   <span class="input-group-addon"><%= fa_icon "envelope" %></span>
   <%= f.input_field :email, class: "form-control" %>
 <% end %>
-<%= f.input :password, label: "Password", wrapper: :vertical_input_group, required: true, hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length) do %>
-  <span class="input-group-addon"><%= fa_icon "lock" %></span>
-  <%= f.input_field :password, class: "form-control" %>
+<!-- Only display password fields when not nested within organization form -->
+<% if @organization.nil? %>
```

Comment:
I really like the underlying idea here, but can we make this an even more explicit check by maybe passing in a variable like "show_password_fields" from the view calling this partial? I'd be worried that we may eventually define `@organization` for a field where we'd still want to hide this

Reply:
Since this view is under the `admin` (Site Admin) namespace, I'm reasonably sure its not being used anywhere else. You should be able to just ditch the password fields entirely. We don't ever want to change a user's password for them.

Reply:
This partial is being used in the edit existing user form, and create new user form, both in the `admin` namespace. If we don't want admins to create passwords for new users, or edit existing user's passwords, I can pull it out! Just want to verify first though, @armahillo.

Reply:
Ah! OK Thanks for checking there! I stand corrected. :+1: